packages/nextjs/services/store/decrypted.ts

```diff
@@ -117,7 +114,8 @@ export const useDecryptValue = <T extends FheTypes>(
   const result = useDecryptedStore(state => state.decryptions[value?.toString() ?? ""]);
 
   useEffect(() => {
-    if (result != null || value == null) return;
+    if (value == null) return;
+    if (result != null && result.value != null) return;
```

Comment:
This is the line that fixes the stuck decryption issue. The result was always null, but now it refetches if the value is null rather than the whole result object